foundry/applications/playerList.d.ts

```diff
@@ -1,27 +1,50 @@
-// @TODO:
-
 /**
  * The active Player List application
  */
 declare class PlayerList extends Application {
   /**
-   * Assign the default options which are supported by the PlayerList UI
+   * @defaultValue
+   * ```typescript
+   * {
+   *   ...super.defaultOptions,
+   *   id: 'players',
+   *   template: 'templates/user/players.html',
+   *   popOut: false
+   * }
+   * ```
    */
   static get defaultOptions(): Application.Options;
 
+  /** @override */
+  render(force?: boolean, options?: Application.RenderOptions): this;
+
+  /** @override */
+  getData(options?: Application.RenderOptions): PlayerList.Data | Promise<PlayerList.Data>;
+
+  /** @override */
+  activateListeners(html: JQuery): void;
+
   /**
-   * Add a context menu to the players UI which allows players to control or release Actors that they own
+   * Return the default context options available for the Players application
    */
-  activateListeners(html: JQuery): any;
+  private _getUserContextOptions(): ContextMenu.Item[];
 
   /**
-   * Prepare the default data which is required to render the PlayerList ui
+   * Toggle display of the Players hud setting for whether or not to display offline players
    */
-  getData(): object;
+  private _onToggleOfflinePlayers(event: Event): void;
```

Comment:
The event type here should be `JQuery.ClickEvent`

Reply:
I made this change